src/simulation/camera.cpp

```
@@ -5,34 +5,34 @@ Camera::Camera(const glm::ivec2& window_sz, double yaw, double pitch,
                DynamicObject* target_obj)
     : radius_(radius), world_up_(world_up), target_obj_(target_obj) {
 
-  yaw_ = yaw;
+  yaw_   = yaw;
```

Clang (14.0.6)

[error] 'yaw_' should be initialized in a member initializer of the constructor

(cppcoreguidelines-prefer-member-initializer,-warnings-as-errors)